ts/Extensions/Breadcrumbs.ts

```diff
@@ -1097,6 +1028,65 @@ if (!H.Breadcrumbs) {
     });
 }
 
+/* *
+ *
+ *  Prototype Properties
+ *
+ * */
+
+interface Breadcrumbs {
+    breadcrumbsGroup: SVGElement;
+    breadcrumbsList: Array<Array<any>>;
+    chart: Chart;
+    isDirty: boolean;
+    isTreemap?: boolean;
+    level: number;
+    options: BreadcrumbsOptions;
+}
+interface BreadcrumbsOptions {
+    buttonTheme: SVGAttributes;
+    buttonSpacing?: number;
+    events?: BreadcrumbsButtonsEventsOptions;
+    floating?: boolean;
+    format?: string;
+    formatter?: BreadcrumbsButtonsFormatter;
+    position?: BreadcrumbsAlignOptions;
+    separator?: SeparatorOptions;
+    showFullPath?: boolean;
+    style?: CSSObject;
+    useHTML?: boolean;
+    zIndex?: number;
+}
+interface BreadcrumbsAlignOptions {
+    align?: AlignValue;
+    verticalAlign?: VerticalAlignValue;
+    alignByTranslate?: boolean;
+    x: number;
+    y: number;
+}
+interface BreadcrumbsClickCallbackFunction {
+    (e: Event, breadcrumbs: Breadcrumbs): (boolean|undefined);
+}
+interface BreadcrumbsButtonsEventsOptions {
+    click?: BreadcrumbsClickCallbackFunction;
+}
+interface BreadcrumbsButtonsFormatter {
+    (
+        breadcrumb: Array<number|null|Point|SeriesType|SVGAElement>,
+        breadcrumbs: Breadcrumbs
+    ): (string);
+}
+interface SeparatorOptions {
+    symbol?: string;
+    size?: number;
+}
```

Comment:
Except for Breadcrumbs interface the rest should go into a `namespace Breadcrumbs`, to be accessible with the default export.